.gitignore

```diff
@@ -4,3 +4,8 @@
 .coverage
 __pycache__
 build
+.vscode
+.env
+venv
+env
+.pytest_cache
```

Comment:
One picky comment here. There's a blank line missing at the end

Reply:
Done